config.py

```diff
@@ -51,11 +55,16 @@ def __init__(
         hide_time_to_first_response: bool,
         ignore_user: List[str],
         labels_to_measure: List[str],
+        enable_mentor_count: str,
+        min_mentor_comments: str,
+        max_comments_eval: str,
+        heavily_involved_cutoff: str,
         search_query: str,
     ):
         self.gh_app_id = gh_app_id
         self.gh_app_installation_id = gh_app_installation_id
         self.gh_app_private_key_bytes = gh_app_private_key_bytes
+        self.search_query = search_query
```

Comment:
already on line 80 below

Reply:
Likely a residual of me fixing mering errors. Removed.